examples/continuous_normalising_flow.ipynb

```diff
@@ -748,7 +749,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.7"
+   "version": "3.10.4"
+  },
+  "vscode": {
+   "interpreter": {
+    "hash": "f2e7013ef9ffca27b94c39f5eba58524f9e175f876d312983be1369a16a681ac"
+   }
```

Comment:
I admire your keenness to rerun all the notebooks to be sure they still run, but I'd rather have a smaller diff to review -- it's much harder to review changes to images in `.ipynb` files.

IMO for little changes it's totally fine to just edit the JSON structure of the code directly. (The pre-commit hooks should verify at least basic checks like syntactic well-formedness of the code produced.)

Reply:
Makes sense, done.